testdata/serve.txt

```diff
@@ -7,6 +7,7 @@ openapi-httprequest --outputdir server --server api.yaml
 
 # Check that we can build the resulting code.
 # TODO check that it actually runs and behaves as expected.
+# TODO(osama-kassem): This test does not work, fix it.
```

Comment:
Isn't this a problem? Please open a ticket on this repo instead, and add a card in JIRA for us not to lose track of it.

Reply:
My change didn't affect it, it was not working.
Will create a ticket & a JIRA card.